src/services/layerHelper.js

```diff
@@ -92,7 +93,7 @@ ngeo.LayerHelper.prototype.createWMTSLayerFromCapabilitites = function(
 
       // Add styles from capabilities as param of the layer
       var layers = result['Contents']['Layer'];
-      var l = goog.array.find(layers, function(elt, index, array) {
+      var l = ol.array.find(layers, function(elt, index, array) {
```

Comment:
yes but unfortunately not available for IE (es6) :
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find